arrow-core/src/main/kotlin/arrow/core/extensions/const/divide/ConstDivideInstance.kt

```diff
@@ -233,19 +282,17 @@ fun <O, A, B, C, D, E, FF, G, H, I> Kind<Kind<ForConst, O>, Tuple8<A, B, C, D, E
 @Deprecated(
   "@extension kinded projected functions are deprecated",
   ReplaceWith(
-  "product(MO, arg1)",
-  "arrow.core.product"
+    "Cons.mapN(MA, this, arg1) { a, b -> Tuple2(a, b) }",
+    "arrow.core.Cons"
```

Comment:
```suggestion
    "arrow.core.Cons", "arrow.core.Tuple2"
```

Missing imports for `product` methods